Lib/fileinput.py

```diff
@@ -259,6 +259,13 @@ def __next__(self):
             # repeat with next file
 
     def __getitem__(self, i):
+        import warnings
+        warnings.warn(
+            "FileInput's __getitem__ method is deprecated. "
```

Comment:
The end user likely uses indexing instead of direct calling of `__getitem__`. What about "Indexing FileInput is deprecated."?

Reply:
How about "Indexing of a FileInput object is deprecated." or "Support for indexing a FileInput object is deprecated."

@vadmium do you have a suggestion on wording of the deprecation message?